docs/src/heat_equation.md

```diff
@@ -0,0 +1,217 @@
+# [Tutorial: Solution of the heat equation](@id tutorial-heat-equation)
+
+Similar to the
+[tutorial on linear advection](@ref tutorial-linear-advection),
+we will demonstrate how to solve a conservative production-destruction
+system (PDS) resulting from a PDE discretization and means to improve
+the performance.
+
+
+## Definition of the production-destruction system
+
+Consider the heat equation
+
+```math
+\partial_t u(t,x) = \mu \partial_x^2 u(t,x),\quad u(0,x)=u_0(x),
+```
+
+with ``μ ≥ 0``, ``t≥ 0``, ``x\in[0,1]``, and homogeneous Neumann boundary conditions.
+We choose the classical central finite difference discretization of the
+Laplacian, resulting in the ODE
+
+```math
+\partial_t u(t) = L u(t),
+\quad
+L = \frac{\mu}{\Delta x^2} \begin{pmatrix}
+    -1 & 1 \\
+    1 & -2 & 1 \\
+    & \ddots & \ddots & \ddots \\
+    && 1 & -2 & 1 \\
+    &&& 1 & -1
+\end{pmatrix}.
+```
+
+The system can be written as a conservative PDS with production terms
+
+```math
+\begin{aligned}
+&p_{i,i-1}(t,\mathbf u(t)) = \frac{\mu}{\Delta x^2} u_{i-1}(t),\quad i=2,\dots,N, \\
+&p_{i,i+1}(t,\mathbf u(t)) = \frac{\mu}{\Delta x^2} u_{i-1}(t),\quad i=1,\dots,N-1,
+\end{aligned}
+```
+
+and destruction terms ``d_{i,j} = p_{j,i}``.
+
+
+## Solution of the production-destruction system
+
+Now we are ready to define a `ConservativePDSProblem` and to solve this
+problem with a method of
+[PositiveIntegrators.jl](https://github.com/SKopecz/PositiveIntegrators.jl) or
+[OrdinaryDiffEq.jl](https://docs.sciml.ai/OrdinaryDiffEq/stable/).
+In the following we use ``N=100`` nodes and the time domain ``t\in[0,1]``.
+Moreover, we choose the initial condition
+
+```math
```

Comment:
😨 `u0` contains negative values! 

Why does it work at all? Seems that positive elements will stay positive and negative elements will stay negative. 

I'd use a nonnegative `u0` here.